base/uk.ac.stfc.isis.ibex.opis/check_opi_info.py

```diff
@@ -0,0 +1,42 @@
+import os
+import win32api
+import xml.etree.ElementTree as ET
+import unittest
+
+DEFAULT_ROOT_DIR = os.path.join(os.path.dirname(__file__), "resources")
+
+
+class TestOpiInfo(unittest.TestCase):
+    def check_path_valid(self, path_str):
+        full_path = os.path.join(DEFAULT_ROOT_DIR, path_str)
+        # check path exists
+        if not os.path.exists(full_path):
+            return ["opi_info.xml error: {} was not found".format(path_str)]
+
+        # check path has no windows-style backslashes
+        if '\\' in path_str:
+            return ['opi_info.xml error: "{}" contains windows-style backslash (\\)!'.format(path_str)]
+
+        # check file case sensitivity on xml and system
+        filesystem_path = win32api.GetLongPathName(win32api.GetShortPathName(full_path)).split("resources\\")[1]
+        if path_str != filesystem_path:
+            return ['opi_info.xml error: info path does not match filesystem path - "{}", should be "{}"'
+                    .format(path_str, filesystem_path)]
+
+        return []
+
+    def test_opi_info(self):
+        file_path = os.path.join(DEFAULT_ROOT_DIR, "opi_info.xml")
```

Comment:
Since this is a constant, it would be more conventional to declare it at the top of the file like:

```python
OPI_INFO_PATH = os.path.join(DEFAULT_ROOT_DIR, "opi_info.xml")
```